src/index.jsx

```diff
@@ -317,6 +318,7 @@ export default class Neuroglancer extends React.Component {
       brainMapsClientId,
       eventBindingsToUpdate,
       callbacks,
+      ngServer,
```

Comment:
@tingzhao this needs to be added to the propTypes

Reply:
as does onSelectionDetailsStateChanged

Reply:
changes uploaded